buildscripts/BuildScripts.csproj

```diff
@@ -1,10 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I was hoping this could serve as package source for things like NUnit.Console. It supports the build but does not really belong in any project. What do you think? 

It also doubles as a conventional include project for anything script related that would be handy to edit directly from within the solution. 

Open to suggestions on naming, any thoughts?